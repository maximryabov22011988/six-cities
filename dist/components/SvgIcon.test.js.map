{"version":3,"file":"SvgIcon.test.js","sourceRoot":"","sources":["../../src/views/components/SvgIcon.test.tsx"],"names":[],"mappings":";;AAAA,+BAA+B;AAC/B,gDAAgD;AAEhD,uCAAgC;AAEhC,EAAE,CAAC,gDAAgD,EAAE,GAAG,EAAE;IACxD,MAAM,IAAI,GAAG,QAAQ;SAClB,MAAM,CACL,oBAAC,iBAAO,IACN,IAAI,EAAC,WAAW,EAChB,MAAM,EAAC,IAAI,EACX,KAAK,EAAC,IAAI,GACV,CACH,CAAC,MAAM,EAAE,CAAC;IAEb,MAAM,CAAC,IAAI,CAAC,CAAC,eAAe,EAAE,CAAC;AACjC,CAAC,CAAC,CAAC"}