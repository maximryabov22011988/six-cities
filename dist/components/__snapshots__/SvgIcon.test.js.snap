// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SvgIcon correctly render 1`] = `
Array [
  <svg
    height="10"
    width="10"
  >
    <use
      xlinkHref="#icon-test"
    />
  </svg>,
  <span
    className="visually-hidden"
  />,
]
`;
